server/api.ts

```diff
@@ -43,6 +43,10 @@ export class Request implements APIRequest {
     return this.#req.method
   }
 
+  get host(): string {
```

Comment:
maybe it should be `hostname` that removes the port?

Reply:
yes I will rename it to `hostname`

Reply:
nice! thanks

Reply:
@ije fixed

Reply:
<img width="180" alt="Screenshot 2021-04-14 at 00 49 24" src="https://user-images.githubusercontent.com/2883484/114590458-6313ee00-9cbb-11eb-9467-b03e5ee7e912.png">

@getspooky  we should strip the **port** as `hostname`.

Reply:
@ije we can simply use `(this.#req.conn.remoteAddr as Deno.NetAddr).hostname`

Reply:
great!